### DISCUSSION TITLE
FB 6591 neues Image flashen geht - bootet immer wieder zu alten Image

### DISCUSSION THREAD

**Discussion by MartMet** [Freetz-NG/freetz-ng#953] [Allgemein] [answered]
Hallo zusammen, 

ich hab hier eine FB6591 (lgi) mit FRITZ!OS: [07.29]

Ich hab mittels Ubuntu VM ein neues Image gebaut und mit
tools/push_firmware geflasht. 

Das flashen dauert ziemlich lange (~128KB/s) lief aber anstandslos durch. Nach einem erneuten PowerReset der FB bootet allerdings immer wieder das alte FRITZ OS 7.29

Muss ich zum switchen des Bootloaders irgendwas beachten? Finde in der UI auch keine Option zum Upgraden auf FRITZ OS 7.57.
Muss ich sonst noch irgendwas beachten?

**Comment by MartMet** [comment, marked as answer]
Nach genauerer Inspektion schien diese FB Vodafone gebrandet zu sein. 

Die richtigen Stichworte für google "7591 debranding".

Hab nach der Anleitung den Bootloader erneuert und die frisch kompilierte Firmware geflasht. Freetz-NG läuft jetzt super. :)

**Comment by fda77** [comment]
Ich glaube "debranden" geht doch nicht mehr, sondern nur wenn es "provider additive" sind. Aber im IPPF wirst du dazu mehr Hilfe bekommen

**Reply by Conan179** [reply]
Bei mir ging es nur, als ich es im Bootloader geändert hatte.

**Reply by fda77** [reply]
Bei "neueren" Geräten ist dies "SETENV firmware_version" aber nicht mehr möglich da die Variable schreibgeschützt ist. Welche geräte genau es sind weiss ich nicht

**Reply by Conan179** [reply]
Uff ich glaube bei mir war es so, das die 6591 zwar die option gesetzt hat, aber ignoriert hat.

**Reply by fda77** [reply]
Ich kann nur sagen dass ich an einer 7520 von 1:1 das branding im Bootloader nicht ändern konnte

**Reply by Conan179** [reply]
Einen kleinen moment bitte! Nicht das wir aneinader vorbei sprechen, ich meinte nicht per  befehl im bootloader es zu ändern, sonden den bootloader per befehl aus zu gelesen, hex editiert und dann wieder geflasht.

**Reply by fda77** [reply]
Tud mir leid, ich hab keine Ahnung was so möglich ist. Hab das nur kurz mal irgend wo überflogen. Ich weiss nicht mal ob der Bootloader selbst überschrieben wird (glaube ich nicht, wenn man den kaputt macht ist das gerät wirklich gebrickt) oder nur die config-partition, wie es ein avm-recovery macht
